console/console-init/ui/src/graphql-module/queries/Address.ts

```diff
@@ -6,6 +6,7 @@
 import gql from "graphql-tag";
 import { ISortBy } from "@patternfly/react-table";
 import { removeForbiddenChars } from "utils";
+import { getCompplexFilterByPattern, getSimpleFilterByPattern } from "./query";
```

Comment:
Can you please revisit the naming convention here?

Reply:
can you please suggest some names so that i can get some help.